pom.xml

```diff
@@ -327,6 +327,7 @@ org.netbeans.html.boot.impl:org.netbeans.html.boot.fx:org.netbeans.html.context.
                     <instructions>
                         <Export-Package>${publicPackages},META-INF.services.*;-noimport:=true;-split-package:=first</Export-Package>
                         <Bundle-SymbolicName>${bundleSymbolicName}</Bundle-SymbolicName>
+                        <Automatic-Module-Name>${bundleSymbolicName}</Automatic-Module-Name>
```

Comment:
OK, we have a new attribute in `MANIFEST.MF` file. However I'd like to see a test (e.g. example) what this is good for.

Please note that there is also `Export-Package` attribute. If we really want to support JDK9, then we shall `export` these packages and make the rest inaccessible. Can you modify the build script to also properly specify exported packages?

Reply:
Hi Jaroslav,

This change is meant only to provide stable automatic module names in case users will use this as a dependency in their JDK9 application, anticipating on making the code modular later on. It appears to me that you want to expand this scope. That's fine, but not trivial and not what I signed up for.
I have very limited time and I have no experience with osgi, felix or html4j. So I don't think I am the right guy for the tasks you are asking.
I am not sure what is tested with the integration tests you request. Would those not test the workings of Java9 and / or the maven-bundle-plugin? I think such a test is a lot of effort (for me to do, given my experience) in relation to this change. I rather spend my time on E.g. the re-licensing or JUnit5 support or something for Netbeans IDE.

Kind regards, 
Dave